packages/dom-helper/lib/prop.js

```diff
@@ -15,18 +15,17 @@ export function normalizeProperty(element, attrName) {
     // TODO should this be an o_create kind of thing?
     cache = {};
     for (key in element) {
-      key = key.toLowerCase();
+      let lowerKey = key.toLowerCase();
       if (isSettable(element, key)) {
-        cache[key] = key;
+        cache[lowerKey] = key;
       } else {
-        cache[key] = UNDEFINED;
+        cache[lowerKey] = UNDEFINED;
       }
     }
     propertyCaches[tagName] = cache;
   }
 
-  // presumes that the attrName has been lowercased.
-  var value = cache[attrName];
+  var value = cache[attrName.toLowerCase()];
```

Comment:
I need @mixonic to chime in why before this "presumed" attrName had been lowercased, when in fact it never was (at least in my testing). From what I could tell, the cache always missed when the `attrName !== attrName.toLowerCase()`. Was that for some reason intended?